src/stream_processor/CMakeLists.txt

```diff
@@ -6,15 +6,20 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 find_package(FLEX REQUIRED)
 find_package(BISON REQUIRED)
 
-add_subdirectory(parser)
+if (${FLEX_FOUND} AND ${BISON_FOUND})
```

Comment:
FLEX and BISON are both required according to lines 6 and 7?

Reply:
Oops. Good catch! :+1: